src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/TextController.cs

```diff
@@ -81,12 +81,11 @@ public async Task<ActionResult<TextResource>> Create(string org, string app, [Fr
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [Produces("application/json")]
-        [Authorize]
```

Comment:
Will removing the authorize attribute allow for anonymous access? I think that is fine, but it is different from resources on the ApplicationController atm. Could perhaps add the AllowAnonymous attribute as well.